05-onchain.md

```diff
@@ -89,21 +90,29 @@ trigger any action.
 # Commitment Transaction
 
 The local and remote nodes each hold a *commitment transaction*. Each of these
-commitment transactions has four types of outputs:
+commitment transactions has six types of outputs:
```

Comment:
> _Up to_ six types of outputs

?

Reply:
Why the question mark? The types are listed below

Reply:
Ah, add _up to_. Added